Controllers/ErrorController.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+
+namespace TravelTrack_API.Controllers;
+
+///<summary>
+/// handles incoming errors (when not in Development)
+///</summary>
+[ApiController]
+public class ErrorController : ControllerBase
```

Comment:
I don't think you'll need a controller like this for a API. Your errors should be returned in the API call that causes the problem. Maybe you have other plans for this controller though.

Reply:
removed unneeded error controller